### PR TITLE
Update paraswap.yaml: Adding Base

### PR DESCRIPTION
**Pull request by mounibec**
Adding Base

### CODE REVIEW

**Review by ccerv1 — CHANGES_REQUESTED** [review]
hi there
address looks good! 
please just add the tags below the network, eg:

```
networks:
  - base
tags:
  - contract
  - factory
```

**Comment by mounibec** [comment]
> hi there address looks good! please just add the tags below the network, eg:
> 
> ```
> networks:
>   - base
> tags:
>   - contract
>   - factory
> ```

Done!

File: data/projects/p/paraswap.yaml [hunk 1]
```diff
@@ -7,6 +7,9 @@ blockchain:
  - address: "0x59C7C832e96D2568bea6db468C1aAdcbbDa08A52"
```
Comment (ccerv1) [inline data/projects/p/paraswap.yaml:7]: Sorry @mounibec - I missed this, but your need to indent the address. YAML is very fussy about indentations 😓

**Review by ccerv1 — CHANGES_REQUESTED** [review]
Sorry @mounibec - I missed this, but your need to indent the address. YAML is very fussy about indentations 😓

**Comment by mounibec** [comment]
> Sorry @mounibec - I missed this, but your need to indent the address. YAML is very fussy about indentations 😓

Missed that too, fixed!

**Review by ccerv1 — APPROVED** [review]

**Comment by ccerv1** [comment]
All good now! Merging. Thanks so much 🚀

**Pull request merged**